src/libs/actions/IOU.ts

```diff
@@ -7973,6 +8001,34 @@ function unholdRequest(transactionID: string, reportID: string) {
         },
     ];
 
+    // If we are holding from the search page, we optimistically update the snapshot data that search uses so that it is kept in sync
```

Comment:
```suggestion
    // If we are unholding from the search page, we optimistically update the snapshot data that search uses so that it is kept in sync
```